### DISCUSSION TITLE
Can we make an html input file with the "hidden" attribute?

### DISCUSSION THREAD

**Discussion by slorion194** [spatie/laravel-html#202] [General]
Hi,

I try to reproduce this :

`<input
	type="file"
	id="upload"
	class="account-file-input"
	hidden
	accept="image/png, image/jpeg"
/>`

I have this but attribute "hidden" missing : 
`{{ html()->input('file','upload', null)->class('account-file-input')->accept('image/png, image/jpeg') }}`

How i can do this? 

Thanks in advance

**Comment by bskl** [comment]
You can do like this;

```php
{{ html()->file('upload')->class('account-file-input')->attribute('hidden')->accept('image/png, image/jpeg') }}
```
and output is this;

```html
<input class="account-file-input" type="file" name="upload" id="upload" hidden accept="image/png, image/jpeg">
```

**Comment by slorion194** [comment]
It's work.

Thanks a lot
